battleship.py

```diff
@@ -62,7 +62,14 @@ def mousePressed(data, event, board):
 Returns: 2D list of ints
 '''
 def emptyGrid(rows, cols):
-    return
+    grid=[]
+    for i in range(rows):
+        col = []
+        for j in range(cols):
+            col.append(EMPTY_UNCLICKED)
+        grid.append(col)
+    return (grid)
+

```

Comment:
@Charan230199 There is no code related to the function createShip. Please check and commit changes to this branch by adding the code.